cardano_node_tests/tests/test_delegation.py

```diff
@@ -629,13 +632,17 @@ def test_undelegate(
         )
 
         # files for deregistering / re-registering stake address
+        stake_address_amount = common.get_conway_address_deposit(cluster_obj=cluster)
```

Comment:
Please change the variable name to something more fitting, like `address_deposit`.